### ISSUE TITLE
523. Continuous Subarray Sum - typos/grammar that are hard to follow

### ISSUE THREAD

**Issue by agatagruza** [LeetCode-Feedback/LeetCode-Feedback#705]
#### Your LeetCode username

gordonek

#### Category of the bug
- [ ] Question
- [x] Solution
- [ ] Language

#### Description of the bug

Approach 3: Using HashMap has multiple bad grammar and misspelled words. For example:
`upto` --> ***up to***

`The processing done is taking the modulus of the the sum upto the i th index with the given k. ` 
--> ***In a HashMap sum%k is a key and i th index is a value.***

`Now, assume that the given sum value at the i th index be equal to rem` 
--> ***Let's assume the sum at the i th index has the value rem***

`Whenever we find a new sum value, which isn't present in the HashMap already, we make an entry in the HashMap of the form, (sum`. The entire formula is missing.  
--> Should be ***map.put(sum, i);***

`Now, assume that the given sum value at the i th index be equal to rem.` 
--> ***is equal to***

`From this observation, we come to the conclusion that ...` 
-->***...we've come to the conclusion that...***

#### Expected behavior

Implementing provided suggestions will fix typos and bad grammar. It will make instructions easier to follow.

**Comment by lc-ivan** [comment]
Hi @agatagruza  ,
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by lc-ivan** [comment]
Hello @agatagruza !
Thank you once again for the feedback regarding this problem. After review, our team plans to rewrite the solution descriptions but we will not be awarding LeetCoins for this ticket. Happy LeetCoding!
